packages/toolpad-app/src/toolpad/AppEditor/PageEditor/RenderPanel/RenderOverlay.tsx

```diff
@@ -268,6 +268,51 @@ export function deleteOrphanedLayoutNodes(
   return draftDom;
 }
 
+const rowColumnCounts: Record<NodeId, number> = {};
```

Comment:
Looks good if it works! We could probably reset this variable if the page changes to keep it from growing too big, but no need to do it yet probably.